flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```

Comment:
Can you provide aarch64 packages?

Reply:
The package is built using GitHub actions which only lets you use x86_64 architecture. I would like to support arm, but I'm just not sure how I would do that until github adds arm support to action runners.

Reply:
```suggestion
  "only-arches": ["x86_64"]
```